superset-frontend/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -91,6 +97,30 @@ const defaultProps = {
   instantFiltering: false,
 };
 
+const StyledFilterContainer = styled.div`
+  display: flex;
```

Comment:
[No need to use theme the hard way](https://github.com/apache/superset/wiki/Emotion-Styling-Guidelines-and-Best-Practices#no-need-to-use-theme-the-hard-way) 😉